ocp_resources/datavolume.py

```diff
@@ -341,6 +346,7 @@ def delete(self, wait=False, timeout=TIMEOUT_4MINUTES, body=None):
         """
         # if garbage collector is enabled, DV will be deleted after success
         if self.exists:
+            warn(gc_deprecatio_warning)
```

Comment:
If DV exists - then garbage collection is not enabled. I think you can just remove it from here. And then see if we can remove the dv delete() function in the 4.19